Simplified/Reader2/LCP/LCPAuthenticationViewController.swift

```diff
@@ -0,0 +1,214 @@
+//
+//  LCPAuthenticationViewController.swift
+//  r2-testapp-swift
+//
+//  Created by Mickaël Menu on 01.03.19.
+//
+//  Copyright 2019 Readium Foundation. All rights reserved.
+//  Use of this source code is governed by a BSD-style license which is detailed
+//  in the LICENSE file present in the project repository where this source code is maintained.
+//
+
+#if LCP
+
+import SafariServices
+import UIKit
+import ReadiumLCP
+
+
+protocol LCPAuthenticationDelegate: AnyObject {
+    
+    func authenticate(_ license: LCPAuthenticatedLicense, with passphrase: String)
+    func didCancelAuthentication(of license: LCPAuthenticatedLicense)
+
+}
+
+class LCPAuthenticationViewController: UIViewController {
+    
+    weak var delegate: LCPAuthenticationDelegate?
+    
+    @IBOutlet weak var scrollView: UIScrollView!
```

Comment:
I'll add some documentation, just a bit of clarification what it is: the delegate is our `LCPLibraryService`, and in this case it conforms to `LCPAuthenticating` protocol. Whenever readium-lcp-swift needs to get authentication passphrase for the license, it calls authentication delegate - `LCPLibraryService` - that creates this view controller.

Reply:
sure, that's good info but that probably belongs on the LCP Service side?

Here i just meant the various IBOutlets, e.g. what is the "hint" for, or what's the different between prompt or message? Just an example. A short doc comment for each outlet would suffice here, since it's a little hard to visualize, also because I don't think we have something equivalent for Adobe DRM, right? 

Actually that brings out a question: why do we need to authenticate for LCP books? I assume this UX has been validated by Product/Design?

Reply:
That's right, Adobe DRM doesn't have this passphrase feature. The view is going to be reviewed today (9 Nov.), added comments to each outlet.